reedsolomon.go

```diff
@@ -16,6 +16,7 @@ import (
 	"errors"
 	"io"
 	"sync"
+	"unsafe"
```

Comment:
Sorry, I cannot allow changes that imports unsafe, without at least being in a separate file, so it can be disabled with tags.